scene/gui/line_edit.cpp

```diff
@@ -1816,6 +1820,7 @@ void LineEdit::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_placeholder_alpha"), &LineEdit::get_placeholder_alpha);
 	ClassDB::bind_method(D_METHOD("set_cursor_position", "position"), &LineEdit::set_cursor_position);
 	ClassDB::bind_method(D_METHOD("get_cursor_position"), &LineEdit::get_cursor_position);
+	ClassDB::bind_method(D_METHOD("get_scroll_offset"), &LineEdit::get_window_pos);
```

Comment:
You wouldn't guess that `window_pos` is actually the scroll position, that might be worth renaming in the C++ code eventually.

Reply:
It would be easy to change as it's internal to `LineEdit`. Do you think it could be part of this PR, or is it too much noise?

Reply:
I think it could be made in this PR directly, there's a good number of occurrences in the file but it's trivial enough that it won't obscure the actual binding change.

Reply:
Done in the latest push.